enforcer-rules/src/site/apt/dependencyConvergence.apt.vm

```diff
@@ -157,3 +164,16 @@ and
         </excludes>
       </dependencyConvergence>
 +---------------------------------------------
+
+* Reporting Only Mode
+
+  By default, when convergence error is detected the build will fail.
+  If you want to have only reporting, without errors in the build, then <<<fail>>> property must be set to <<<false>>>.
+
++---------------------------------------------
+      <dependencyConvergence>
+        <fail>false</fail>
```

Comment:
Each rule has already a parameter - `level` - you can set it with value `WARN` and should be what you need

Reply:
Good to know, this part I missed. I will remove this config.

What about scopes, is it ok?

Reply:
for scope will be ok - it is good idea
please also add IT test 

scope - import is used only for dependency management - not for dependency

It will be new feature - so jira issue will be required

Reply:
@slawekjaranowski, I have removed a part about reporting-only mode, only scopes left. Could you please let me know if changes are ok, and could you please assist me with JIRA issue? Thank you!